### ISSUE TITLE
Adding picture to MP3 intermittently works

### ISSUE THREAD

**Issue by scott-lin** [Zeugma440/atldotnet#170]
## The problem

I'm adding a `Front` picture to an `mp3` and it intermittently succeeds.

Coincidentally, it actually succeeds exactly 50% of the time. I have a jpg in the same folder as my audio file, and my code (see below) tries to add the `PictureInfo` to the files embedded pictures. What I'm doing to repro, is actually just running my console app subsequent times. I can see every other time the image is added. I am clearing the embedded pictures list each attempt.

Based on the logs below from ATL, it seems to be deallocating the space required for the image if it already is present on the file.

**Side question**: is clearing the picture list to remove all existing images a misuse of the library? Should I remove all images another way?

## Environment

* ATL version (or git revision) that exhibits the issue: 4.13.0
* Last ATL version that did not exhibit the issue (if applicable):
* OS/version used to run ATL: Windows 10

## Details

Note, my image is 190 KB.

**Execution attempt 1 -> Succeeds when no prior image is embedded**. This attempt results in the image being embedded properly.
ATL log:
```
========================================
    Found 1 regions
    #0 : 0->3331(3.25 KB) IsBufferable = True
    ========================================
    ------------ REGION 0
    ------ ZONE default@0
    Allocating 3.25 KB
    newTagSize : 193.26 KB
    Disk stream operation (direct) : Lengthening (delta=190.01 KB)
```

**Execution attempt 2 -> Fails when image is already embedded** This attempt results in the image being removed and not being replaced (see my code).
ATL log:
```
========================================
    Found 1 regions
    #0 : 0->197903(193.26 KB) IsBufferable = True
    ========================================
    ------------ REGION 0
    ------ ZONE default@0
    Allocating 193.26 KB
    newTagSize : 3.25 KB
    Disk stream operation (direct) : Shortening (delta=-190.01 KB)
```

The next execution attempt succeeds, then fails, then succeeds, and so on with the same logs above.

## Code To Reproduce Issue

```
var track = new Track( audioFilePath );
PictureInfo coverArtInfo = PictureInfo.fromBinaryData( File.ReadAllBytes( coverArtFilePath ), PictureInfo.PIC_TYPE.Front );
track.EmbeddedPictures.Clear();
track.EmbeddedPictures.Add( coverArtInfo );

var log = new AtlLog();
track.Save();
foreach ( ATL.Logging.Log.LogItem item in log.Items )
{
    Console.WriteLine( item.Message );
}
```

Let me know if you need more details. Thanks.

**Comment by Zeugma440** [comment]
I'll get back to you soon. Real life is kinda demanding at the moment 😆

**Comment by Zeugma440** [comment]
The issue has been reproduced. The fix has been done and will be available for next release

**Comment by Zeugma440** [comment]
The fix is available on today's v4.14

**Comment by scott-lin** [comment]
Thank you!
